### ISSUE TITLE
Small adjustment to link "How to Meet WCAG 2.1"

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag21#958]
At: https://www.w3.org/TR/WCAG21/#wcag-2-1-supporting-documents there's a link refering to a "How to Meet WCAG 2.1" document.

This is actually a link to the "How to Meet WCAG 2" document WITHOUT the ".1"

In the filter you could choose 2.1 as wel as 2.0 or 2.1 addition only, but the document itself serves both.

The link therefore should be [How to Meet WCAG 2](https://www.w3.org/WAI/WCAG21/quickref/)

**Comment by michael-n-cooper** [comment]
This issue was moved to w3c/wcag#412
